### ISSUE TITLE
About evaluation...

### ISSUE THREAD

**Issue by Jctrp** [woodfrog/maptracker#29]
Hi, thank you for this excellent open-source work!

I am a beginner in the field of BEV online mapping and am currently learning about this project.
I would like to ask if there is an online test server available for submitting results for parallel comparison, specifically for nuScenes? 
Also, during inference and evaluation, is the test set used? (I noticed that only xx_val.pkl is used in the settings.)

**Comment by woodfrog** [comment]
Thanks for your interest. There were some benchmarks hosted online, like [this](https://github.com/Tsinghua-MARS-Lab/Online-HD-Map-Construction-CVPR2023).

Right, we use the val split for inference and evaluation like all previous works. I believe the ground truth of the test set is not publicly available (not 100% sure about this).

**Comment by Jctrp** [comment]
Thank you for your reply!  
As a beginner, I am still debugging and learning the training steps of this work and have not yet reproduced the performance reported in the paper.  
So, may I ask incidentally, can the metrics reported in the paper be obtained through the 'Inference and evaluation' steps in this project?(Especially with the nuScenes dataset, as AV2 is somewhat large for my current setup.)

**Comment by Jctrp** [comment]
Wishing you smooth progress in your work and looking forward to your new research achievements.
